### ISSUE TITLE
Event Listener on .card with .card-reveal only works with a onClick event.

### ISSUE THREAD

**Issue by bajahranks** [Dogfalo/materialize#6533]
Issue:
Cards can't be revealed via a keypress which is a huge accessibility issue. It works fine with a click of the mouse but it doesn't work with a keypress. Also cards cannot be navigated via tab by default, this needs to be added manually. 

Expected Outcome:
On keypress (Enter) card reveals (activated) when the .card-reveal class is used.

Possible Solution:
Add a keypress eventListener to .card-reveal. I believe it uses the code block below...
`function(e) {
  if (t(this).find("> .card-reveal").length) {
    var i = t(e.target).closest(".card");
    void 0 === i.data("initialOverflow") && i.data("initialOverflow", void 0 === i.css("overflow") ? "" : i.css("overflow")), t(e.target).is(t(".card-reveal .card-title")) || t(e.target).is(t(".card-reveal .card-title i")) ? t(this).find(".card-reveal").velocity({
      translateY: 0
    }, {
      duration: 225,
      queue: !1,
      easing: "easeInOutQuad",
      complete: function() {
        t(this).css({
          display: "none"
        }), i.css("overflow", i.data("initialOverflow"))
      }
    }) : (t(e.target).is(t(".card .activator")) || t(e.target).is(t(".card .activator i"))) && (i.css("overflow", "hidden"), t(this).find(".card-reveal").css({
      display: "block"
    }).velocity("stop", !1).velocity({
      translateY: "-100%"
    }, {
      duration: 300,
      queue: !1,
      easing: "easeInOutQuad"
    }))
  }
}`
